clvm/operators.py

```diff
@@ -43,22 +55,19 @@ class OperatorDict(dict):
 
     def __new__(class_, d: Dict):
         self = super(OperatorDict, class_).__new__(class_, d)
-        self.unknown_op_handler = self.unknown_op_raise
+        self.unknown_op_handler = default_unknown_op
         return self
 
     def __call__(self, op: bytes, arguments: CLVMObject) -> Tuple[int, CLVMObject]:
         f = self.get(op)
         if f is None:
-            def f(args):
-                return self.unknown_op_handler(op, args)
```

Comment:
I thought this code wasn't as straight-forward as it could be.

Reply:
+1